UPGRADE.md

```diff
@@ -1,5 +1,27 @@
 # Upgrade notes
 
+## v5 to v6
+
+Bake definition has been split into two files, so you can include either tags
+or labels or both:
+
+```yaml
+      -
+        name: Build
+        uses: docker/bake-action@v3
+        with:
+          files: |
+            ./docker-bake.hcl
+            ${{ steps.meta.outputs.bake-file-tags }}
+            ${{ steps.meta.outputs.bake-file-labels }}
+          targets: build
+```
+
+> **Note**
+> 
+> `bake-file` output is still available for backward compatibility, but it's
+> recommended to use `bake-file-tags` and `bake-file-labels` outputs instead.
```

Comment:
Why is this recommended? I would more describe it as "alternatively, you can use the individual files that only set tags or labels".

Reply:
Removed this section and added note in the README for the alternative syntax:

![image](https://github.com/docker/metadata-action/assets/1951866/bf4b72b9-c41d-481f-8833-716153718749)

Reply:
It still suggests multiple files as a recommended path, what I don't understand as it is more complicated way to use the action.

Reply:
This was to avoid users getting confused if they want to set annotations in follow-up with #352 (see https://github.com/crazy-max/docker-metadata-action/tree/annotations-alt#annotations) but that seems right to still make `bake-file` the goto behavior. Updated to keep current syntax.